tests/Doctrine/Tests/Common/Annotations/CachedReaderTest.php

```diff
@@ -157,53 +157,39 @@ public function testAvoidCallingFilemtimeTooMuch() : void
         /* @var $cache Cache|MockObject */
         $cache = $this->createMock('Doctrine\Common\Cache\Cache');
 
-        // first pass => cache ok for method 1
-        // we load annotations AND filemtimes for this file
         $cache
-            ->expects($this->at(0))
+            ->expects($this->exactly(6))
             ->method('fetch')
-            ->with($this->equalTo($cacheKeyMethod1))
-            ->will($this->returnValue([$route1])); // Result was cached, but there was an annotation;
-        $cache
-            ->expects($this->at(1))
-            ->method('fetch')
-            ->with($this->equalTo('[C]' . $cacheKeyMethod1))
-            ->will($this->returnValue($cacheTime));
+            ->withConsecutive(
+                // first pass => cache ok for method 1
+                // we load annotations AND filemtimes for this file
+                [$this->equalTo($cacheKeyMethod1)],
```

Comment:
Does somebody know the difference between using this and using `[$cacheKeyMethod1]`?